client/src/js/indexes/components/History.js

```diff
@@ -1,10 +1,19 @@
 import { map, sortBy } from "lodash-es";
 import React from "react";
-import { Col, ListGroup, ListGroupItem, Row } from "react-bootstrap";
 import styled from "styled-components";
-import { LoadingPlaceholder, Panel } from "../../base";
+import { LoadingPlaceholder, BoxGroupSection, BoxGroupHeader, BoxGroup } from "../../base";
 
-const StyledRebuildHistoryEllipsis = styled(ListGroupItem)`
+const HistoryItem = styled(BoxGroupSection)`
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+`;
+
+const Content = styled.div`
```

Comment:
Rename to `RebuildHistoryContent`. These names have to be descriptive for debug purposes.